go.mod

```diff
@@ -11,3 +11,5 @@ require (
 	gopkg.in/check.v1 v1.0.0-20190902080502-41f04d3bba15 // indirect
 	gopkg.in/yaml.v2 v2.2.7
 )
+
+go 1.13
```

Comment:
Can this be removed? It's causing the Travis failure. I'll clean this up in another PR.

Reply:
Oh yes, I meant to remove that before pushing, cleaned up now.